<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddTeacherWidget</class>
 <widget class="QWidget" name="AddTeacherWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="returnButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>40</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>主菜单</string>
   </property>
  </widget>
  <widget class="QLabel" name="idLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>54</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>学号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>54</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>姓名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="mathLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>数学分析：</string>
   </property>
  </widget>
  <widget class="QLabel" name="enLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>四级英语：</string>
   </property>
  </widget>
  <widget class="QLabel" name="cLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>200</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>C++：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="idLineEdit">
   <property name="geometry">
    <rect>
     <x>182</x>
     <y>60</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入学号：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameLineEdit">
   <property name="geometry">
    <rect>
     <x>182</x>
     <y>100</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入姓名：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="mathLineEdit">
   <property name="geometry">
    <rect>
     <x>182</x>
     <y>130</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入成绩：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="enLineEdit">
   <property name="geometry">
    <rect>
     <x>182</x>
     <y>170</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入成绩：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cLineEdit">
   <property name="geometry">
    <rect>
     <x>182</x>
     <y>210</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入成绩：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>270</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>270</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>260</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;PS：删除只需输入学号&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
